internal/integrationtest/util.go

```diff
@@ -211,6 +207,9 @@ func CallUntilSuccess(t *testing.T, rawClient raw.Client, interval time.Duration
 
 // Call sends an echo request to the client.
 func Call(ctx context.Context, rawClient raw.Client) error {
+	if ctx == nil {
```

Comment:
Yeh, and linter complained so I had to pass a background. This is not exercised anymore.